### PR TITLE
Renamed 'AI Platform' and 'AI Platform (Unified)' to 'Vertex AI'

### PR DESCRIPTION
**Pull request by ivanmkc**
Renamed:
- 'AI Platform' -> 'Vertex AI'
- 'AI Platform (Unified)' -> 'Vertex AI'
- 'AI Platform Notebooks' -> 'Google Cloud Notebooks'
- 'com/ai-platform-unified' -> 'com/vertex-ai' # For links

### CODE REVIEW

File: ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb [hunk 1]
```diff
@@ -42,7 +42,7 @@
       "source": [
         "# Explainable AI via MB SDK on Custom Tabular model\n",
         "\n",
-        "To use this Jupyter notebook, copy the notebook to an AI Platform(Unified) Notebooks instance with Tensorflow installed and open it. You can run each step, or cell, and see its results. To run a cell, use Shift+Enter. Jupyter automatically displays the return value of the last line in each cell. For more information about running notebooks in AI Platform(Unified) Notebook, see the [AI Platform(Unified) Notebook guide](https://cloud.google.com/ai-platform-unified/docs/general/notebooks).\n",
+        "To use this Jupyter notebook, copy the notebook to an Vertex AI(Unified) Notebooks instance with Tensorflow installed and open it. You can run each step, or cell, and see its results. To run a cell, use Shift+Enter. Jupyter automatically displays the return value of the last line in each cell. For more information about running notebooks in Vertex AI(Unified) Notebook, see the [Vertex AI(Unified) Notebook guide](https://cloud.google.com/vertex-ai/docs/general/notebooks).\n",
```
Comment (sasha-gitg) [inline ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb:45]: In some places AI Platform(Unified) was replaced with Vertex AI(Unified).

File: ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb [hunk 2]
```diff
@@ -56,7 +56,7 @@
         "id": "-VL5avdUJ7-V"
       },
       "source": [
-        "### Install AI Platform (Unified) SDK\n",
+        "### Install Vertex AI SDK\n",
```
Comment (sasha-gitg) [inline ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb:59]: "Vertex AI SDK" should be "Vertex SDK for Python" throughout these notebooks.

**Review by sasha-gitg — DISMISSED** [review]
What's the feasibility of changing the file names as well.

ie:
`AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb` to `Vertex_SDK_for_Python_Explainable_AI_Custom_Tabular.ipynb` 

I know some of these may have references outside of this repo so we could do this as a three step process. Create a copy with a Vertex name, change external references, and delete the notebooks with AI Platform in the name. 

This can be done in the same folder:
`ai-platform-unified/notebooks/unofficial/sdk/Vertex_SDK_for_Python_Explainable_AI_Custom_Tabular.ipynb`
or in different folder (my preference):
`vertex-ai/notebooks/unofficial/sdk/Vertex_SDK_for_Python_Explainable_AI_Custom_Tabular.ipynb`

@alecglassford for input.

Thanks!

File: ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb (hunk 1 above)
Comment (ivanmkc) [reply ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb:45]: Good catch, the space seems missing. Will check for that.

File: ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb (hunk 2 above)
Comment (ivanmkc) [reply ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb:59]: Thanks, will do that.

**Comment by ivanmkc** [comment]
@sasha-gitg @alecglassford This PR is meant to be non-linking breaking.

I had talked with @thedriftofwords before about changing the folders/filenames and we agreed to do that when we move to a new repo, which @morgandu is planning soon. That way, we only need to break the links once.

File: ai-platform-unified/notebooks/unofficial/pipelines/control_flow_kfp.ipynb [hunk 3]
```diff
@@ -249,9 +249,9 @@
         "\n",
         "1. [Make sure that billing is enabled for your project](https://cloud.google.com/billing/docs/how-to/modify-project).\n",
         "\n",
-        "1. [Enable the AI Platform (Unified), Cloud Storage, and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com,compute_component,storage-component.googleapis.com). \n",
+        "1. [Enable the Vertex AI, Cloud Storage, and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com,compute_component,storage-component.googleapis.com). \n",
         "\n",
-        "1. Follow the \"**Configuring your project**\" instructions from the AI Platform Pipelines documentation.\n",
+        "1. Follow the \"**Configuring your project**\" instructions from the Vertex AI Pipelines documentation.\n",
```
Comment (amygdala) [inline ai-platform-unified/notebooks/unofficial/pipelines/control_flow_kfp.ipynb:254]: Just "Vertex Pipelines". 
Same for another instance below.

File: ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb [hunk 4]
```diff
@@ -251,9 +251,9 @@
         "\n",
         "1. [Make sure that billing is enabled for your project](https://cloud.google.com/billing/docs/how-to/modify-project).\n",
         "\n",
-        "1. [Enable the AI Platform (Unified), Cloud Storage, and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com,compute_component,storage-component.googleapis.com). \n",
+        "1. [Enable the Vertex AI, Cloud Storage, and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com,compute_component,storage-component.googleapis.com). \n",
         "\n",
-        "1. Follow the \"**Configuring your project**\" instructions from the AI Platform Pipelines documentation.\n",
+        "1. Follow the \"**Configuring your project**\" instructions from the Vertex AI Pipelines documentation.\n",
```
Comment (amygdala) [inline ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb:256]: "Vertex Pipelines". Same for another instance below.

File: ai-platform-unified/notebooks/unofficial/pipelines/pipelines_intro_kfp.ipynb [hunk 5]
```diff
@@ -878,14 +878,14 @@
       "source": [
         "### Create a service account (quick start version)\n",
         "\n",
-        "See the documentation for more details on the process of creating and configuring a service account to work with AI Platform Pipelines. In brief: \n",
+        "See the documentation for more details on the process of creating and configuring a service account to work with Vertex AI Pipelines. In brief: \n",
         "\n",
         "- Go to the Cloud Console -> IAM & Admin -> Service Accounts, click “**CREATE SERVICE ACCOUNT**”. \n",
```
Comment (amygdala) [inline ai-platform-unified/notebooks/unofficial/pipelines/pipelines_intro_kfp.ipynb:883]: In PR #341 , I changed this section (which was no longer correct) and just swapped in a link to the docs.  So there may be a merge conflict here. 😬 (sorry).
I don't _think_ any of my other changes on that PR will cause a conflict, but something to watch out for.

**Review by amygdala — COMMENTED** [review]
I didn't remark on all the instances of this, but there are a number of places where we will need to change "Vertex AI Pipelines" --> "Vertex Pipelines".  I can do this separately after this main PR is in if that would make things easier.

File: ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb (hunk 2 above)
Comment (ivanmkc) [reply ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb:59]: Done

File: ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb (hunk 1 above)
Comment (ivanmkc) [reply ai-platform-unified/notebooks/unofficial/sdk/AI_Platform_(Unified)_SDK_Explainable_AI_Custom_Tabular.ipynb]: Done

File: ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-online.ipynb [hunk 6]
```diff
@@ -78,11 +78,11 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this notebook, you create a custom-trained model from a Python script in a Docker container using the AI Platform (Unified) client library, and then do a prediction on the deployed model by sending data. Alternatively, you can create custom-trained models using `gcloud` command-line tool, or online using the Cloud Console.\n",
+        "In this notebook, you create a custom-trained model from a Python script in a Docker container using the Vertex AI client library, and then do a prediction on the deployed model by sending data. Alternatively, you can create custom-trained models using `gcloud` command-line tool, or online using the Cloud Console.\n",
         "\n",
         "The steps performed include:\n",
         "\n",
-        "- Create an AI Platform (Unified) custom job for training a model.\n",
+        "- Create an Vertex AI custom job for training a model.\n",
```
[the remark this answers is not in the document]
Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-online.ipynb:85]: done

File: ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-batch.ipynb [hunk 7]
```diff
@@ -98,11 +98,11 @@
         "\n",
         "This tutorial uses billable components of Google Cloud (GCP):\n",
         "\n",
-        "* AI Platform (Unified)\n",
+        "* Vertex AI\n",
         "* Cloud Storage\n",
         "\n",
-        "Learn about [Cloud AI Platform\n",
-        "pricing](https://cloud.google.com/ai-platform-unified/pricing) and [Cloud Storage\n",
+        "Learn about [Cloud Vertex AI\n",
```
[the remark this answers is not in the document]
Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-batch.ipynb:104]: done

File: ai-platform-unified/notebooks/official/vizier/gapic-vizier-multi-objective-optimization.ipynb [hunk 8]
```diff
@@ -53,7 +53,7 @@
       "source": [
         "# Optimizing multiple objectives\n",
         "\n",
-        "This tutorial demonstrates [AI Platform(Unified) Vizier](https://cloud.google.com/ai-platform-unified/docs/vizier/overview) multi-objective optimization. Multi-objective optimization is concerned with mathematical optimization problems involving more than one objective function to be optimized simultaneously\n",
+        "This tutorial demonstrates [Vertex AI Vizier](https://cloud.google.com/vertex-ai/docs/vizier/overview) multi-objective optimization. Multi-objective optimization is concerned with mathematical optimization problems involving more than one objective function to be optimized simultaneously\n",
```
[the remark this answers is not in the document]
Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/vizier/gapic-vizier-multi-objective-optimization.ipynb:56]: Thanks! Done.

File: ai-platform-unified/notebooks/unofficial/pipelines/control_flow_kfp.ipynb (hunk 3 above)
Comment (ivanmkc) [reply ai-platform-unified/notebooks/unofficial/pipelines/control_flow_kfp.ipynb:254]: done

File: ai-platform-unified/notebooks/unofficial/pipelines/pipelines_intro_kfp.ipynb (hunk 5 above)
Comment (ivanmkc) [reply ai-platform-unified/notebooks/unofficial/pipelines/pipelines_intro_kfp.ipynb:883]: I may check your PR and replicate the changes in this PR to fix it.

**Review by ivanmkc — COMMENTED** [review]
I've found the remaining names that are prefixed with 'Vertex AI':

```
Vertex AI API
Vertex AI API Client
Vertex AI APIs
Vertex AI Administrator
Vertex AI AutoML
Vertex AI AutoML Vision
Vertex AI Custom Code
Vertex AI Custom Training
Vertex AI Dataset
Vertex AI Experiments
Vertex AI ML Metadata
Vertex AI Managed Dataset
Vertex AI Managed Text
Vertex AI Model Monitoring
Vertex AI Prediction
Vertex AI Predictions
Vertex AI Python
Vertex AI Text Dataset
Vertex AI Training
Vertex AI User
Vertex AI Video Dataset
```

Let me know if any of these need changing.

File: ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-online.ipynb [hunk 9]
```diff
@@ -497,9 +497,9 @@
         "id": "import_aip"
       },
       "source": [
-        "#### Import AI Platform (Unified) client library\n",
+        "#### Import Vertex AI client library\n",
```
Comment (ooriana) [inline ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-online.ipynb:500]: Official name for the python client lib is now "Vertex SDK for Python".

File: ai-platform-unified/notebooks/unofficial/pipelines/pipelines_intro_kfp.ipynb (hunk 5 above)
Comment (ivanmkc) [reply ai-platform-unified/notebooks/unofficial/pipelines/pipelines_intro_kfp.ipynb:883]: For pipelines, I saw conflict but just used your changes as I assume you already switched out the names/links.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**